mne/filter.py

```diff
@@ -1389,3 +1389,62 @@ def _get_filter_length(filter_length, sfreq, min_length=128, len_x=np.inf):
         if not isinstance(filter_length, integer_types):
             raise ValueError('filter_length must be str, int, or None')
     return filter_length
+
+
+class FilterMixin(object):
+    """Object for Epoch/Evoked filtering"""
+
+    def filter(self, method='savgol', h_freq=None):
+        """Filter the data
+
+        Parameters
+        ----------
+        method : str
```

Comment:
I think for now I'll make it so there is no `method` param since there's only one method anyway.